src/book/01-team/02-joining/01-how-we-work.mdx

```diff
@@ -10,7 +10,7 @@ The NUbots development process is a combination of in-person and remote collabor
 
 ## Meetings and Lab Work
 
-We hold weekly meetings on Zoom at **Wednesday from 1-2pm**. This time may change based on team members' availabilities.
+We hold weekly meetings in the NUbots lab at **Friday 4-5pm**, with the option to call in via Zoom. The meeting time may change based on team members' availabilities.
```

Comment:
I feel like we should pull this out to an environment variable somehow so we don't need to keep committing changes every time our meeting time changes... This is okay for now tho

Reply:
It does kinda go in our general update of NUbook each semester or so (as per pinned message on the documentation channel) so it's probably fine since we'll be needing to update current members list about as frequently too.